lang/ruby/test/test_logical_types.rb

```diff
@@ -124,6 +124,70 @@ def test_bytes_decimal
     end
   end
 
+  def test_bytes_decimal_default
+    Avro.disable_field_default_validation = true
```

Comment:
once https://github.com/apache/avro/pull/2275 is merged, we can rebase and remove this hack

Reply:
The previous PR has been merged.

Reply:
rebased and this hack is removed as well

ac4a2b5afe0c8f0fb9e04ecdedb0ca5cb13cdb1d